docs/docs.md

```diff
@@ -491,14 +513,17 @@ Anyone can view any user
 ```ts
 type Data = {
   username: string
-  name: string
+  name: {
```

Comment:
I don't really like this... can we just do firstName and lastName?

Reply:
:heavy_check_mark: